src/errors.ts

```diff
@@ -49,7 +50,9 @@ export function wrapApiError(maybeError: Error, message?: string): Error {
   const info = {
     req: error.request
       ? formatRequest({...error.config, ...error.request})
-      : formatRequest(error.config),
+      : error.config
+      ? formatRequest(error.config)
+      : undefined,
```

Comment:
it complains `error.config` might be `undefined`.
I don't know if we can just do 
```
req: formatRequest({...error.config, ...error.request})
 ```
does bad thing happen when you do `...` `undefined`?

Reply:
try it out on https://www.typescriptlang.org/play

Reply:
yea i did :))) 
i might just have to use some interface to assign `undefined` value. let me try again 😭

Reply:
i think it works